### DISCUSSION TITLE
is it possible to use Casadi framework but dont use the solver inside?

### DISCUSSION THREAD

**Discussion by ziqiang-sha** [casadi/casadi#3753] [Q&A]
export the optimization problem to a standard form that another solver can use. CasADi allows exporting the problem to formats. is that possible?

**Comment by charleskhazoom** [comment]
Yes, you can use all of casadi's autodiff capabilities without using any solver. There is a blogpost about it for using matlab's fmincon but it works with any solver : https://web.casadi.org/blog/fmincon/
